### ISSUE TITLE
Add support for printing DOM Elements to `jest-emotion`

### ISSUE THREAD

**Issue by rszewczyk** [emotion-js/emotion#619]
- `emotion` version: `9.1.0`
- `react` version: `16.3.0`

Relevant code.

```javascript
import React from 'react'
import { render } from 'react-testing-library'
import { createSerializer } from 'jest-emotion'
import * as emotion from 'emotion'
import { Something } from './stuff'

expect.addSnapshotSerializer(createSerializer(emotion))

it('snapshots dom nodes', () => {
  const { container } = render(<Something>hello!</Something>)
  expect(container.firstChild).toMatchSnapshot()
})

```

As expected, this doesn't produce a snapshot with styles and replaced selectors because `react-testing-library` returns DOM elements, not React elements. 

Suggested solution:

Add support for printing styles/replaced selectors for DOM Element snapshots. I'd like to submit a PR for this. I have a [proof of concept here](https://gist.github.com/rszewczyk/4375363cddff6636262d76d8a073b860). What I'm not sure about is some of the implementation details. Should, it be implemented as a separate serializer or an enhancement to the current serializer? If the latter, should be enabled via configuration, always enabled, etc?

Thank in advance for your help/feedback!

**Comment by mitchellhamilton** [comment]
@rszewczyk Thanks for starting work on this!!

I think we should add it to the current serializer and have it enabled by default but add a `dom` option that will disable it when the option is false.

**Comment by rszewczyk** [comment]
Thanks @mitchellhamilton! I'll work on this today.

**Comment by rszewczyk** [comment]
@mitchellhamilton [PR opened](https://github.com/emotion-js/emotion/pull/622)

**Comment by mitchellhamilton** [comment]
Closed via #622
